onnx/defs/math/defs.cc

```diff
@@ -1069,6 +1069,69 @@ ONNX_OPERATOR_SET_SCHEMA(
         .SetContextDependentFunctionBodyBuilder(BuildContextDependentFunctionBodyClip)
         .TypeAndShapeInferenceFunction(propagateShapeAndTypeFromFirstInput));
 
+std::function<void(OpSchema&)>
+SoftmaxFamilyDocGenerator(const char* name, const char* description, const char* equation) {
```

Comment:
Is this from merging some other PR? I am a bit mystified why this is showing up in this PR, from where.

Reply:
I moved this from utils from here because I think it belongs to this file. I was updating utils to move the "get value from tensor" helper in.

Reply:
Oh, I see. The day we update any op in this (Softmax) family and move the old one to old.cc, we will need it in the shared utils file. But I am fine either way.

Reply:
Right - for now it's only used in a single place so I moved it to where it is used.